godot-bindings/src/godot_version.rs

```diff
@@ -5,7 +5,9 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
-//#![allow(unused_variables, dead_code)]
+// This file is explicitly included in unit tests
+// while all the functions included are used only with `custom-api` and `custom-api-json` features.
+#![allow(unused_variables, dead_code)]
```

Comment:
Could this be more limited, i.e.
- if only some items need it, add the attribute there instead of on the file
- if only some cfgs need it, use `#[cfg_attr(...)]`